java-maps-mapsplatformdatasets/.repo-metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "api_shortname": "maps-mapsplatformdatasets",
+  "name_pretty": "Maps Platform Datasets API",
+  "product_documentation": "https://developers.google.com/maps/documentation",
```

Comment:
This parent product homepage is a placeholder for the product documentation link which is not yet available, but our script requires something starting with "https://".